classes/CartRule.php

```diff
@@ -170,10 +170,10 @@ class CartRuleCore extends ObjectModel
             'date_upd' => ['type' => self::TYPE_DATE, 'validate' => 'isDate'],
             /* Lang fields */
             'name' => [
-                'type' => self::TYPE_STRING,
+                'type' => self::TYPE_HTML,
```

Comment:
```suggestion
                'type' => self::TYPE_STRING,
```

It should be a string, not an HTML input

Reply:
No. With `TYPE_STRING`, the value is strip_tagged. So all characters are not allowed.